tests/FakeObject.php

```diff
@@ -3,6 +3,8 @@
 
 class FakeObject
```

Comment:
Did you missed to extend stdClass in this case. Or we are all good with just the 2 properties.

Reply:
Add properties fix the issue. see https://github.com/auraphp/Aura.Sql/pull/219

Reply:
I noticed that PR after my comment. Cool.